addons/rmsmartshape/lib/TesselationVertexMapping.gd

```diff
@@ -0,0 +1,35 @@
+extends RefCounted
```

Comment:
Better stick with the same file naming convention as in the rest of the code base, which is snake_case.